### PR TITLE
Add pack_unpack utils and unit tests

### PR DESCRIPTION
**Pull request by tmarkovski**
Adds pack/unpack support. Unit tests are disabled since there's no docker image with 1.8 support yet, until libindy tags 1.8 as rc.
Can be run locally.

**Related to**: #72

### CODE REVIEW

File: src/AgentFramework.Core/Utils/CryptoUtils.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+namespace AgentFramework.Core.Utils
+{
+    internal class CryptoUtils
+    {
+        /// <summary>Packs a message</summary>
+        /// <param name="wallet">The wallet.</param>
+        /// <param name="recipientKey">The recipient key.</param>
+        /// <param name="senderKey">The sender key.</param>
+        /// <param name="message">The message.</param>
+        /// <returns></returns>
```
Comment (tplooker) [inline src/AgentFramework.Core/Utils/CryptoUtils.cs]: Specify the return types on all methods?

File: src/AgentFramework.Core/Utils/CryptoUtils.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        public static async Task<T> UnpackAsync<T>(Wallet wallet, byte[] message)
+        {
+            var result = await Crypto.UnpackMessageAsync(wallet, message);
+            var unpacked = result.ToObject<UnpackResult>();
+            return unpacked.Message.ToObject<T>();
+        }
+    }
+
+    public class UnpackResult
```
Comment (tplooker) [inline src/AgentFramework.Core/Utils/CryptoUtils.cs]: Comments?

**Comment by tplooker** [comment]
Looks good, just added a couple of suggestions.

**Comment by tmarkovski** [comment]
I added comments. The Utils and the Result class are marked as internal. Perhaps this is temporary, I wanted to make sure that what we expose through the library doesn't require devs to use pack/unpack themselves.

**Pull request merged**